txredisapi.py

```diff
@@ -2380,14 +2381,19 @@ def __init__(self, isLazy=False, handler=ConnectionHandler):
 
 
 def makeConnection(host, port, dbid, poolsize, reconnect, isLazy,
-                   charset, password, connectTimeout, replyTimeout,
+                   charset, password, ssl_context_factory, connectTimeout, replyTimeout,
                    convertNumbers):
     uuid = "%s:%d" % (host, port)
     factory = RedisFactory(uuid, dbid, poolsize, isLazy, ConnectionHandler,
                            charset, password, replyTimeout, convertNumbers)
     factory.continueTrying = reconnect
     for x in range(poolsize):
-        reactor.connectTCP(host, port, factory, connectTimeout)
+        if isinstance(ssl_context_factory, bool) and ssl_context_factory is True:
```

Comment:
if `ssl_context_factory is True`, it is guaranteed to be a bool, so this can be simplified:

```suggestion
        if ssl_context_factory is True:
```